cmd/create/cluster/cmd.go

```diff
@@ -604,6 +608,24 @@ func init() {
 	)
 	flags.MarkHidden("billing-account")
 
+	flags.StringVar(
+		&args.clusterAdminUser,
+		"cluster-admin-user",
+		"",
+		"Username of Cluster Admin.\n"+
+			"Username must not contain /, :, or %%",
+	)
+
+	flags.StringVar(
+		&args.clusterAdminPassword,
+		"cluster-admin-password",
+		"",
+		"Password of Cluster Admin.\n"+
+			"The password must\n"+
+			"- Be at least 14 characters (ASCII-standard) without whitespaces\n"+
+			"- Include uppercase letters, lowercase letters, and numbers or symbols (ASCII-standard characters only)",
```

Comment:
NP: I think backticks would be useful here, they would eliminate the need for `\n`  and extra quotes for every newline you make

Reply:
I would write an example, but i cant figure out how to make " ` " show up twice without doing `this` :(

Reply:
thanks for the comment, updated to use backticks